__mocks__/azure-maps-control.js

```diff
@@ -1,3 +1,11 @@
+class DataSource {
```

Comment:
let's follow our current way to mock, or it is necessary to have class right here?

Reply:
the change was needed for below discussed instanceof check to pass

Reply:
for the current way of mocking, mockConstructor produces the plain object, if we going to rely on instanceof check to differentiate between DataSource and VectorTileSource, probably needs a change, do you know a more elegant way for doing this rather then that-kind class declaration?

Reply:
Im also agree with that to stay with current way fo mocking until we find better way.

Reply:
@psrednicki: you mean the original way? things like `dataSourceRef instanceof atlas.source.DataSource` will return false and break the tests, that's basically the only reason I added class declaration

Reply:
Ok, got your point now, its just a mock so Im okey with that. @msasinowski maybe you have any thought of that?

Reply:
We can use a prototype chain or use just something like `Object.create(XXX)`, I'm good with that but can you create a new file inside the __mocks__ e.g. data_source.mock.js and put this right there?